packages/app/features/edit-profile/uploadProfilePictureBanner.tsx

```diff
@@ -0,0 +1,10 @@
+import { Text, View } from "app/design-system";
+import React from "react";
+
+export function UploadProfilePictureBanner() {
```

Comment:
do we need a separate component for it?

Reply:
Probably not, but I would argue that this code is easier to read.

Reply:
In my opinion, it's so small that there's no point in separating it.